netket/driver/abstract_variational_driver.py

```diff
@@ -75,25 +75,25 @@ def _forward_and_backward(self):
         Returns:
             the update for the weights.
         """
-        self._forward()
-        dp = self._backward()
-        return dp
+        self._forward()  # pragma: no cover
+        dp = self._backward()  # pragma: no cover
+        return dp  # pragma: no cover
```

Comment:
This is not covered because it is only a fallback implementation, right?

I think this can be done in a cleaner way like this (without marking every line separately):
```python
    def _forward_and_backward(self):  # pragma: no cover
        ...
```

Reply:
you sure it works?

Reply:
ok, I used it

Reply:
> you sure it works?


I think so, at least based on [these docs](https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html).